src/ejabberd_admin.erl

```diff
@@ -333,6 +346,11 @@ register(User, Host, Password) ->
             {ok, io_lib:format("User ~s@~s successfully registered", [User, Host])}
     end.
 
+generate_user() ->
+    mongoose_bin:join([mongoose_bin:gen_from_timestamp(),
```

Comment:
How about adding `<<"private_jid_">>` at the beginning or/and at the end for easier search??
@mattsaidwhat what do you think?

Reply:
I would not do that preemptively.